examples/html-kitchen-sink/stories/__snapshots__/welcome.stories.storyshot

```diff
@@ -99,3 +99,5 @@ exports[`Storyshots Welcome Welcome 1`] = `
 
 </div>
 `;
+
+exports[`Storyshots Welcome Welcome Async 1`] = `[object Promise]`;
```

Comment:
This is a bit odd?

Reply:
I’m unfamiliar with how to update the storyshots with async stories. Do you know of any examples I could use to update this?

Reply:
Looks like we need to do something in storyshots to properly support this.

Reply:
We could figure it our together in a pair programming session if you'd like?